apps/web/src/app/dashboard/teacher/quiz/_components/NewQuizButton.tsx

```diff
@@ -44,43 +28,45 @@ import {
   CardFooter,
   CardHeader
 } from "@/components/ui/card";
-import { trpc } from "../../../../../utils/trpc/client";
 import { useState } from "react";
-import { TrpcReactQueryOptions } from "../../../../../utils/trpc/lib";
 import Image from "next/image";
 import { Heading } from "src/components/ui";
+import { trpc } from "../../../../../utils/trpc/client";
 import { ScrollArea } from "@/components/ui/scroll-area";
 import { Separator } from "@/components/ui/separator";
 
 interface Props {
-  initialData: TrpcReactQueryOptions["questionBank"]["list"]["initialData"];
+  initialData: {
+    questionsCount: number;
+    _count: {
+      questions: number;
+    };
+    id: number;
+    createdAt: Date;
+    updatedAt: Date;
+    title: string;
+    authorId: string;
+  };
 }
 interface quizProps {
   no_of_questions: number;
-  question_bank_id: number;
   quiz_name: string;
   timer: number;
 }
 export const NewQuizButton = ({ initialData }: Props) => {
+  console.log(initialData);
```

Comment:
can remove

Reply:
Resolved in https://github.com/peiyuans-pals/toohakai/commit/19647d875eb27ab17a412917c24f814f0f436a1f